src/parts/_forms.css

```diff
@@ -43,6 +62,7 @@ textarea {
   width: 100%;
   box-sizing: border-box;
   resize: vertical;
+  min-height: 140px;
```

Comment:
Do we need the `min-height` here?
Initial height for `<textarea>` can be controlled with its `row` attribute, e.g. `<textarea rows="8">`

Reply:
It's a reasonable default that can be overridden with rows, I took the change from #153.

Reply:
Hmm, alright I guess.
Can't be overridden though, anything below `rows="5"` (which gives you the 140px of height) does not have any effect so you can't have a small textarea even if you want to. (without also adding something like `style="min-height: 0"`)

Reply:
We could do 

```css
textarea:not([rows]) {
  min-height: 140px; // use 5 rows as default
}
```
?